CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [8.1.2] - 08-24-2020
```

Comment:
Since we are adding something new this should be `8.2.0` since it isn't a patch.

Reply:
Good point, I'll adjust that.